src/platform/w3dengine/client/w3droadbuffer.cpp

```diff
@@ -2127,178 +2128,182 @@ bool W3DRoadBuffer::Insert_Y(Vector2 loc, int index1, float scale)
 
 void W3DRoadBuffer::Insert_4Way(Vector2 loc, int index1, float scale)
 {
-    TRoadPt *pt1 = nullptr;
-    TRoadPt *pt2 = nullptr;
-    TRoadPt *pt3 = nullptr;
-    TRoadPt *pt4 = nullptr;
-    TRoadPt *pt5 = nullptr;
-    TRoadPt *pt6 = nullptr;
-    TRoadPt *pt7 = nullptr;
-    TRoadPt *pt8 = nullptr;
+
+    TRoadPt *pr1 = nullptr;
```

Comment:
what are pr and pc? might make more sense to give a comment explaining what they represent.

Reply:
no idea, names in BFME2 wb, pc maybe pointer center? really no idea